src/bindings/python/tests/test_direct_comparison/test_shape.py

```diff
@@ -0,0 +1,21 @@
+from openvino import Shape, PartialShape
+
+def test_shape_eq_list():
+    a = [1, 2, 3]
```

Comment:
the same -- please, don't use one-later variables -- neither linters nor developers don't like it :)
Moreover these to tests may be merged. Please check [this doc](https://docs.pytest.org/en/7.1.x/example/parametrize.html)